dms/migrations/12.0.5.0.0/post-migration.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    directories = (
+        env["dms.directory"]
+        .sudo()
+        .search(
+            [("storage_id.save_type", "=", "attachment"), ("group_ids", "!=", False)]
+        )
+    )
+    directories.write({"group_ids": False})
```

Comment:
This is failing when doing an actual migration:

```
2021-04-16 12:36:36,492 1 INFO prod OpenUpgrade: dms: post-migration script called with version 12.0.4.0.7 
2021-04-16 12:36:36,499 1 ERROR prod OpenUpgrade: dms: error in migration script dms/migrations/12.0.5.0.0/post-migration.py: 'project.task' 
2021-04-16 12:36:36,499 1 ERROR prod OpenUpgrade: 'project.task' 
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/site-packages/openupgradelib/openupgrade.py", line 1875, in wrapped_function
    if use_env2 else cr, version)
  File "/opt/odoo/auto/addons/dms/migrations/12.0.5.0.0/post-migration.py", line 15, in migrate
    directories.write({"group_ids": False})
  File "/opt/odoo/auto/addons/dms/models/directory.py", line 662, in write
    res = super(DmsDirectory, self).write(vals)
  File "/opt/odoo/auto/addons/mail/models/mail_thread.py", line 321, in write
    result = super(MailThread, self).write(values)
  File "/opt/odoo/auto/addons/mail/models/mail_activity.py", line 621, in write
    return super(MailActivityMixin, self).write(vals)
  File "/opt/odoo/custom/src/odoo/odoo/models.py", line 3364, in write
    self._write(store_vals)
  File "/opt/odoo/auto/addons/dms/models/dms_security_mixin.py", line 329, in _write
    return super(DmsSecurityMixin, self)._write(vals)
  File "/opt/odoo/custom/src/odoo/odoo/models.py", line 3451, in _write
    self.check_access_rule('write')
  File "/opt/odoo/auto/addons/dms/models/directory.py", line 204, in check_access_rule
    model = self.env[record.res_model]
  File "/opt/odoo/custom/src/odoo/odoo/api.py", line 831, in __getitem__
    return self.registry[model_name]._browse((), self)
  File "/opt/odoo/custom/src/odoo/odoo/modules/registry.py", line 177, in __getitem__
    return self.models[model_name]
KeyError: 'project.task'
```

You must either move this to `end-migration` or avoid ORM usage.